clconf/clconf.go

```diff
@@ -99,6 +106,19 @@ func DecodeBase64Strings(values ...string) ([]string, error) {
 	return contents, nil
 }
 
+func backCompat(result interface{}, err error) (map[interface{}]interface{}, error) {
```

Comment:
function name doesn't really say enough.  think this needs a comment or better name (especially once it is not part of an MR about providing backwards compat with a specific mod and has to stand on its own in the codebase)

Reply:
Fixed.